### ISSUE TITLE
line() got an unexpected keyword argument 'name'

### ISSUE THREAD

**Issue by woaichipinngguo** [facebookresearch/visdom#328]
Is any update about visdom?  An error occurred When I was using the following code snippet in the demo file. But I can use  "viz.updateTrace".    @ynd 
`
# line updates
win = viz.line(
    X=np.column_stack((np.arange(0, 10), np.arange(0, 10))),
    Y=np.column_stack((np.linspace(5, 10, 10),
                        np.linspace(5, 10, 10) + 5)),
        opts=dict(legend=['1','11'])
)
viz.line(
        X=np.arange(21, 30),
        Y=np.arange(1, 10),
        win=win,
        name='1',
        update='append'
    )
TypeError: line() got an unexpected keyword argument 'name'
`

**Comment by JackUrb** [comment]
Hi @woaichipinngguo - What version of visdom are you running? The github docs are describing features that are found by compiling from source, and admittedly the pypi version is lagging behind. You can try installing from source using `pip install -e .`.

**Comment by woaichipinngguo** [comment]
@JackUrb  I installed it with conda.  Is it outdated? can you give me more details about install the newest version? I can't understand `installing from source using pip install -e ..`. Thanks!!!

**Comment by JackUrb** [comment]
Yeah the current version that conda will pull is 0.1.7, where the demo file displays functionality of the most recent version (0.1.7.2 - progress towards 0.1.8) which is what is in the repo. If you run `pip install -e .` or `easy_install .` from within the visdom directory, it should update you to the most recent version.
